tune_sklearn/tune_search.py

```diff
@@ -265,6 +268,11 @@ class TuneSearchCV(TuneBaseSearchCV):
             determined by 'Pipeline.warm_start' or 'Pipeline.partial_fit'
             capabilities, which are by default not supported by standard
             SKlearn. Defaults to True.
+        stop_on_plateau (bool|dict|TrialPlateauStopper): Stop trials early if
```

Comment:
Perhaps it would be a good idea to just let users pass their own Stopper instance? That way users could just extend the Stopper class for their own purposes.

Reply:
So you mean instead of these arguments we just support `stopper` or so and document how to pass a `TrialPlateauStopper` for this use case?

Reply:
Yeah, I believe that would be a good idea. That way users could define their own Stoppers, or import other Stoppers from Tune - and we would not need to add special support for each of them in tune-sklearn.

Reply:
I think that makes much sense.